src/elements/content-sidebar/activity-feed/task-new/Assignees.js

```diff
@@ -0,0 +1,113 @@
+        const hiddenAssigneeCount = assigneeCount - maxAvatars;
+        const overflowLabel = `+${hiddenAssigneeCount}`;
+        const visibleAssignees = assignees.entries
+            .slice(0, maxAvatars)
+            .map(({ id: assignmentId, target, status: assigneeStatus }) => {
+                return (
+                    <Tooltip key={assignmentId} text={<AssigneeTooltipLabel user={target} status={assigneeStatus} />}>
+                        <AssigneeStatus
+                            key={assignmentId}
+                            status={assigneeStatus}
+                            user={target}
+                            getAvatarUrl={getAvatarUrl}
+                            data-testid="task-assignment-status"
+                        />
+                    </Tooltip>
+                );
+            });
+        const hiddenAssignees = assignees.entries.map(({ id: assignmentId, target, status: assigneeStatus }) => {
+            const statusMessage = statusMessages[assigneeStatus] || null;
+            return (
+                <div key={assignmentId} className="bcs-task-assignment-list-item">
+                    <AssigneeStatus
+                        status={assigneeStatus}
+                        className="bcs-task-assignment-list-item-avatar"
+                        user={target}
+                        getAvatarUrl={getAvatarUrl}
+                        data-testid="task-assignment-status"
+                    />
+                    <div className="bcs-task-assignment-list-item-details">
+                        <div>{target.name}</div>
+                        <div className="bcs-task-assignment-list-item-status">{statusMessage}</div>
+                    </div>
+                </div>
+            );
+        });
+        return (
+            <div className="bcs-task-assignment-container">
+                <div className="bcs-task-assignments">
+                    {visibleAssignees}
+                    {hiddenAssigneeCount > 0 && (
+                        <Flyout position="top-left">
+                            <button
+                                type="button"
+                                className="bcs-task-assignment-count-container"
+                                data-testid="task-assignment-overflow"
+                            >
+                                <Tooltip text={intl.formatMessage(messages.tasksFeedMoreAssigneesLabel)}>
+                                    <span className="bcs-task-assignment-count bcs-task-assignment-avatar">
+                                        {overflowLabel}
+                                    </span>
+                                </Tooltip>
+                            </button>
+                            <Overlay>
+                                <div className="bcs-task-assignment-list">
+                                    <h5 className="bcs-task-assignment-list-title">
```

Comment:
Is this semantically correct or is it only intended to apply a certain visual style? If the latter, it should be done via CSS while using the appropriate heading level for the heading context.